ciri/api/tests/test_store_transactions.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <unity/unity.h>
+
+#include "ciri/api/api.h"
+#include "ciri/api/tests/defs.h"
+#include "consensus/defs.h"
+#include "consensus/test_utils/bundle.h"
+#include "consensus/test_utils/tangle.h"
+
+static char *test_db_path = "ciri/api/tests/test.db";
+static char *ciri_db_path = "ciri/api/tests/ciri.db";
+static connection_config_t config;
+static iota_api_t api;
+static tangle_t tangle;
+static transaction_validator_t transaction_validator;
+
+void setUp(void) {
```

Comment:
snake case

Reply:
`setUp` and `tearDown` are Unity functions that are respectively called at the beginning and at the end of every test, can't change their name :(

Reply:
Ah, alright